sunpy/map/map_factory.py

```diff
@@ -172,9 +173,16 @@ def _parse_args(self, *args, silence_errors=False, allow_errors=False, **kwargs)
             elif isinstance(arg, str) and is_url(arg):
                 # Replace URL string with a Request object to dispatch on later
                 args[i] = Request(arg)
-            elif possibly_a_path(arg):
+            elif possibly_a_path(arg) and not is_uri(arg):
                 # Replace path strings with Path objects
                 args[i] = pathlib.Path(arg)
+            elif is_uri(arg):
```

Comment:
There is an explicit except in `is_uri` to avoid any paths that start with `http` or `https`, so line 173 and 179 should not conflict. If the path starts with `http`/`https` it will hit line 173.